app/views/partners/family_requests/_list.html.erb

```diff
@@ -1,42 +1,47 @@
+<% if @error_messages %>
+  <div class="alert alert-danger" role="alert">
+    <% @error_messages.each do |message| %>
+      <%= message %><br>
+    <% end %>
+  </div>
+<% end %>
+
 <table class='table table-striped'>
   <thead>
-  <tr>
-    <th>Guardian</th>
-    <th>Child</th>
-    <th>Item Needed</th>
-    <th>Include This Child?</th>
-  </tr>
-  </thead>
-  <tbody class='fields'>
-  <% @children.each_with_index do |child, index| %>
     <tr>
-      <td>
-        <%= child.family.guardian_first_name %>
-        <%= child.family.guardian_last_name %>
-      </td>
-      <td>
-        <%= child.first_name %> <%= child.last_name %>
-      </td>
-      <td>
-        <% if child.item_needed_diaperid %>
-          <%= current_partner.organization.item_id_to_display_string_map[child.item_needed_diaperid] %>
-        <% else %>
-          <%= "N/A" %>
-        <% end %>
-      </td>
-      <td>
-        <div class="custom-control custom-switch form-switch">
-          <% if child.item_needed_diaperid %>
-            <%= check_box_tag "child-#{child.id}", child.active, child.active,
-                              class: "custom-control-input",
-                              id: "child-#{child.id}" %>
-                            <label class="custom-control-label" for="child-<%= child.id %>">
-                              <span class="sr-only">Include This Child?</span>
-                            </label>
-          <% end %>
-        </div>
-      </td>
+      <th>Guardian</th>
+      <th>Child</th>
+      <th colspan="2">Items Needed</th>
     </tr>
-  <% end %>
+  </thead>
+  <tbody class='fields'>
+    <% @children.each_with_index do |child, index| %>
+      <tr>
+        <td><%= child.family.guardian_first_name %> <%= child.family.guardian_last_name %></td>
+        <td><%= child.first_name %> <%= child.last_name %></td>
+        <td colspan="2"> <!-- This colspan allows us to use the space for the 'Item Needed' column -->
+          <table class="table">
+            <% child.items.each do |item| %>
+              <tr>
+                <td>
+                  <%= item.name %>
+                </td>
+                <td style="width: 200px;">
+                  <div class="custom-control custom-switch form-switch">
+                    <%= check_box_tag "children_items[#{child.id}][]", item.id,
+                                      @children_items.fetch(child.id.to_s, []).include?(item.id.to_s),
+                                      class: "custom-control-input",
+                                      id: "include_item_#{child.id}_#{item.id}" %>
```

Comment:
Often best to try to align the ids with the names where possible. A better name might be `children_items_#{child.id}`